pkg/instance/instance.go

```diff
@@ -472,7 +472,7 @@ func FuzzerCmd(args *FuzzerCmdArgs) string {
 		verbosityArg = fmt.Sprintf(" -vv=%v", args.Verbosity)
 	}
 	flags := []tool.Flag{}
-	if args.Slowdown > 0 {
+	if args.Slowdown != 1 {
```

Comment:
What about qemu instances that have Slowdown>1?
Why are ignoring optionalFlags? It feels wrong. But if it's unused, it should be removed.
Why tests did not catch this? Can we make them catch this bugs in future?

Reply:
Hmm, indeed. Seems that we have no way to get rid of the optional args flag, so brought it back.

> Why tests did not catch this? 

Made a small fix to `TestFuzzerCmd`, which could have exposed this problem.

PTAL